server/reportConfiguration/dpsConstants/correctionRequestReasons.ts

```diff
@@ -0,0 +1,16 @@
+// Generated with ./scripts/importDpsConstants.ts 2024-09-11T10:04:11.653Z
+
+export const correctionRequestReasons = [
+  /** Mistake */
+  'MISTAKE',
+  /** Incorrect information */
+  'INCORRECT_INFORMATION',
+  /** Missing information */
+  'MISSING_INFORMATION',
```

Comment:
I think the description should be a field as well, so it's machine readable as well...

Maybe all these constant values should be objects rather than simply strings, this way:
- these files would all look similar
- could be extended for whatever reason (e.g. to add another field)
- they can have things description as a field rather than as a comment